documentation/ingles/aodvv2_process.md

```diff
@@ -0,0 +1,108 @@
+
+In  case  of  link  breaks,  AODVv2  uses  route  error  (RERR)messages  to  notify  affected  nodes,  i.e.,  nodes  that  could potentially use this link: 
+if a link break is detected by a node,it invalidates all routes stored in the node’s own routing table that actually use the broken link. Then it broadcasts a RERR message containing the unreachable destinations to all (direct) neighbours using this route. When updating routing table entries,  nodes use sequence numbers to determine the freshness of an entry. The larger the sequence number, the fresher the information a sequence number with value 0 indicates that the number is not known. To maintain the information about sequence numbers each node  stores  its  own  sequence  number.  It  is a common belief that sequence numbers are sufficient to guarantee loop freedom if they are monotonically increased over time. Whenever a node initiates a route request or a route reply, it increments its own sequence number and transmits the incremented value as part of the message. Whenever a RREQ or a RREP message is forwarded, an intermediate node adds its sequence number together with its IP-address to the content of the message.
+
+## Route Tables
+
+Every received route message contains a route and consequently is evaluated to check for any improvement.
+
+```block
+Note that an rreq message contains a route to its source while an rrep message contains a route to its destination. 
+
+Therefore, as the routes are identiﬁed by their destinations, in the former case, the destination of the route 
+
+is the originator of the message and in the latter, it is the destination of the message
+``` 
```

Comment:
```suggestion
>Note that an rreq message contains a route to its source while an rrep message contains a route to its destination. 
>
>Therefore, as the routes are identiﬁed by their destinations, in the former case, the destination of the route 
>
>is the originator of the message and in the latter, it is the destination of the message
```

I've removed the code block and used quotes so this can be highlighted better